### ISSUE TITLE
How to change "m3u8" to "json" and "ts" to "js" extension

### ISSUE THREAD

**Issue by altunoren** [arut/nginx-rtmp-module#1621]
My question: output file "m3u8", "ts" extensions.

How to change output "mtv.m3u8" file to "mtv.json" "mtv-123122313.ts" file to "mtv-123122313.js" ?

For example: 

mtv.json

```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:1797
#EXT-X-TARGETDURATION:5
#EXTINF:5.000,
mtv-1610887534362.js
#EXTINF:5.000,
mtv-1610887539364.js
#EXTINF:5.000,
mtv-1610887544374.js
```

i want to show my output ts file like mtv-1610887544374.js

in some site i saw same extensions but i cant do same. i think its possible. but how? anyone know about this something?

**Comment by koundinyagoparaju** [comment]
Why do you want to do so?

**Comment by the-poor-gamer-124** [comment]
we need to do so, how could we?

**Comment by altunoren** [comment]
> Why do you want to do so?
Just need. 

> we need to do so, how could we?
i do it.

**Comment by the-poor-gamer-124** [comment]
Did you succed?

**Comment by altunoren** [comment]
Yes bro. i do that. How i can share with you? For to dont share with "Why do you want to do so?" questioner :)))))

**Comment by the-poor-gamer-124** [comment]
Salamus#7176 Discord, we can talk about streaming there

**Comment by altunoren** [comment]
That say you must add me. altunoren#5718

**Comment by hsynclb** [comment]
> My question: output file "m3u8", "ts" extensions.
> 
> How to change output "mtv.m3u8" file to "mtv.json" "mtv-123122313.ts" file to "mtv-123122313.js" ?
> 
> For example:
> 
> mtv.json
> 
> ```
> #EXTM3U
> #EXT-X-VERSION:3
> #EXT-X-MEDIA-SEQUENCE:1797
> #EXT-X-TARGETDURATION:5
> #EXTINF:5.000,
> mtv-1610887534362.js
> #EXTINF:5.000,
> mtv-1610887539364.js
> #EXTINF:5.000,
> mtv-1610887544374.js
> ```
> 
> i want to show my output ts file like mtv-1610887544374.js
> 
> in some site i saw same extensions but i cant do same. i think its possible. but how? anyone know about this something?

Selam,
Ulaşamadım hocam size discord, twitter yazdım ama müsait zamanınızda geri dönebilir misiniz?

**Comment by krimounette** [comment]
> Yes bro. i do that. How i can share with you? For to dont share with "Why do you want to do so?" questioner :)))))

Hello
Man ive add you in discored check your pm thank you

**Comment by krimounette** [comment]
> Salamus#7176 Discord, we can talk about streaming there

hello
did you find any solution for it please?

**Comment by slayer1ss** [comment]
I have the same problem, can anyone contact me at SANANE#3921 discord

**Comment by boomcar69** [comment]
Hello,

Can anyone explain me how to setup this?
I searched alot how to get this done but didn't find any good answer.

I see some sites streaming live streams (www.9goal.se) using this technic.

You can contact me on Discord:   Sportnews#2593

Thanks in advance

**Comment by redspor** [comment]
Merhaba, discord üzerinden iletişime geçtim bilgi verebilir misiniz? uzun süredir araştıryıorum ama sonuç alamadim

**Comment by the-poor-gamer-124** [comment]
hire a developer ! from fiverr

**Comment by andres-rivera-05** [comment]
> Sí, hermano, yo hago eso. ¿Cómo puedo compartir con ustedes? Para no compartir con "¿Por qué quieres hacer eso?" cuestionador :)))))
Hello, can you share with me the information on how you achieved it?
